lib/active_data/model/associations/reflections/embeds_many.rb

```diff
@@ -5,6 +5,11 @@ module Model
     module Associations
       module Reflections
         class EmbedsMany < Base
+          def initialize(*)
+            super
+            owner.attribute name, mode: :association
```

Comment:
Let's probably choose only one attribute definition style, also this will break AR embeded associations since there is no need to declare such an attribute